fog/ledger/server/tests/router_integration.rs

```diff
@@ -68,18 +67,18 @@ fn create_store_config(
         client_auth_token_max_lifetime: Default::default(),
         omap_capacity,
         sharding_strategy: ShardingStrategy::Epoch(EpochShardingStrategy::new(block_range)),
-        poll_interval: Duration::from_millis(250),
+        poll_interval: POLL_INTERVAL,
```

Comment:
when the poll interval was made configurable it got changed from 10ms to 250ms everywhere, even in tests.
Moving this back to 10ms to make the tests run faster

Reply:
For my understanding, the reason for the poll interval being at 250ms is resource responsibility? vs, in the test, it's fine to run faster.